app/src/main/java/org/mozilla/focus/utils/DialogUtils.kt

```diff
@@ -313,40 +313,6 @@ object DialogUtils {
                     .build()
                     .also { it.show() }
 
-    fun showContentServiceOnboardingSpotlight(
-        activity: FragmentActivity,
-        targetView: View,
-        dismissListener: DialogInterface.OnDismissListener,
-        ok: View.OnClickListener?
-    ): Dialog = SpotlightDialog.Builder(activity, targetView)
-            .spotlightConfigs(
-                RectangleSpotlightConfigs(
-                    width = activity.resources.getDimensionPixelSize(R.dimen.content_service_focus_view_width),
```

Comment:
Could we also remove the `content_service_focus_view_width` and `content_service_focus_view_radius` definitions from `dimens.xml`?

Reply:
those two assets are still used in DialogUtils.showContentServiceRequestClickSpotlight